streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/exception/ApiAlertException.java

```diff
@@ -26,18 +28,22 @@
  * ...
  * </pre>
  */
-public class ApiAlertException extends RuntimeException {
+public class ApiAlertException extends AbstractApiException {
 
     public ApiAlertException(String message) {
-        super(message);
+        super(message, ResponseCode.CODE_FAIL_ALERT);
     }
 
     public ApiAlertException(Throwable cause) {
-        super(cause);
+        super(cause, ResponseCode.CODE_FAIL_ALERT);
     }
 
     public ApiAlertException(String message, Throwable cause) {
-        super(message, cause);
+        super(message, cause, ResponseCode.CODE_FAIL_ALERT);
     }
 
+    @Override
+    public String getNotificationMessagePrefix() {
+        return "Alert exception: ";
```

Comment:
oh, I didn't express clearly, `ApiAlertException` is presented directly to the front-end alert, I think there is no need for any prefix, because the message of the alert is already a clear message

Reply:
Thanks for your clarification. Updated.